fortran/tuvx/grid.F90

```diff
@@ -140,6 +151,23 @@ function grid_t_constructor(grid_name, grid_units, number_of_sections, error) &
 
   end function grid_t_constructor
   
+  !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+  
+  integer function number_of_sections(this, error) result( n_sections )
+    use musica_util, only: error_t, error_t_c, dk => musica_dk
+
```

Comment:
Looks like `dk` is not used. Could you remove this?

Reply:
done!